sdk/bootstrap/build.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# exit immediately on failure
+set -e
+
+BOOTSTRAP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_ROOT_DIR=$(dirname $(dirname $BOOTSTRAP_DIR))
+
+if [ -z "$GOPATH" -o -z "$(which go)" ]; then
+  echo "Missing GOPATH environment variable or 'go' executable. Please configure a Go build environment."
+  exit 1
+fi
+
+REPO_NAME=dcos-commons # CI dir does not match repo name
+GOPATH_MESOSPHERE=$GOPATH/src/github.com/mesosphere
```

Comment:
For maximal defensiveness, quote all envvars like `$GOPATH`, and envvars that contain those envvars, that could be set by a user to contain spaces.